src/main/java/org/jboss/forge/arquillian/command/AddAlgeronConsumerDependenciesCommand.java

```diff
@@ -0,0 +1,74 @@
+package org.jboss.forge.arquillian.command;
+
+import org.jboss.forge.addon.facets.FacetFactory;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.context.UIExecutionContext;
+import org.jboss.forge.addon.ui.hints.InputType;
+import org.jboss.forge.addon.ui.input.UISelectOne;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.metadata.WithAttributes;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.result.Results;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+import org.jboss.forge.arquillian.api.TestFrameworkFacet;
+import org.jboss.forge.arquillian.container.model.ContractConsumerLibrary;
+import org.jboss.forge.arquillian.testframework.algeron.AlgeronConsumer;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+
+public class AddAlgeronConsumerDependenciesCommand extends AbstractAlgeronCommand {
+
+   @Inject
+   private FacetFactory facetFactory;
+
+   @Inject
+   private AlgeronConsumer algeronConsumerFacet;
```

Comment:
Facets are better handled if created from FacetFactory and then installing afterwards.

eg. @Inject FacetFactory facetFactory;

and then 

FacetClass fc = facetFactory.create(project, FacetClass.class);
fc.setXXX(...)/
facetFactory.installFacet(project,fc)